treeherder/etl/th_publisher.py

```diff
@@ -31,14 +33,18 @@ def post_treeherder_collections(th_collections):
         logger.info(
             "collection loading request: {0}".format(
                 th_request.get_uri(th_collections[project].endpoint_base)))
-        response = th_request.post(th_collections[project])
-
-        if not response or response.status_code != 200:
-            errors.append({
-                "project": project,
-                "url": th_collections[project].endpoint_base,
-                "message": response.text
-            })
+
+        collection_chunks = th_collections[project].get_chunks(chunk_size)
+
+        for collection in collection_chunks:
+            response = th_request.post(collection)
+
+            if not response or response.status_code != 200:
```

Comment:
now that we are using request we should just use response.raise_for_status if it's not called already by thclient